src/components/Navbar.tsx

```diff
@@ -69,7 +69,7 @@ export const Navbar = ( props: NavbarProps ) => {
         <div className="flex justify-between h-auto py-2 px-10">
           <div className="flex flex-row justify-center items-center gap-4">
             <img
-              src={ `${ isProd ? "/emurr/images/logo_vert_5.png" : "images/logo_vert_5.png" }` }
+              src={ `${ isProd ? "/emurr/images/logo_vert_5.png" : "/images/logo_vert_5.png" }` }
```

Comment:
We are using a urlPath for uniformity now. when merging we can overwrite this change